pageserver/src/config.rs

```diff
@@ -207,6 +207,9 @@ pub struct PageServerConf {
     pub background_task_maximum_delay: Duration,
 
     pub control_plane_api: Option<Url>,
+
+    /// JWT token for use with the control plane API.
+    pub control_plane_api_token: Option<String>,
```

Comment:
it this ok to be logged ?

Reply:
I've just added a `SecretString` type with a redacting `Debug` impl, to avoid this getting printed if the config is dumped somewhere.

We'll have similar questions elsewhere in our deploy, can pick that up on slack.